.gitignore

```diff
@@ -3,3 +3,4 @@ coverage/
 test/coverage/
 .bundle
 bundle
+.tags
```

Comment:
I kept running into this and finally forgot to remove my tags file before pushing. Let me know if you prefer this in a unique PR.